tools/toolchain/scripts/install_libvdwxc.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash -e
+[ "${BASH_SOURCE[0]}" ] && SCRIPT_NAME="${BASH_SOURCE[0]}" || SCRIPT_NAME=$0
+SCRIPT_DIR="$(cd "$(dirname "$SCRIPT_NAME")" && pwd -P)"
+libvdwxc_ver=${libvdwxc_ver:-master}
```

Comment:
The commit sha could be part of the version:
```
libvdwxc_ver=${libvdwxc_ver:-master-429a80027a2ec2c97e2f6d9a3dc84843f2739865}
```

Reply:
- I actually have a problem mpich compiled in the toolchain. Curiously configure does not find the libs despite the detection of the header files. Any idea why it is behaving that way? 

- I will fix this. No worries.